### ISSUE TITLE
Tibial fracture

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5887]
**Preferred term label:** Tibial fracture

**Synonyms** Fracture of the Tibia; Broken tibia

**Definition (free text, please give PubMed ID)** Fracture of the tibia.

**Parent term (use hpo.jax.org/app)** Abnormality of tibia morphology HP:0002992; Bone fracture HP:0020110; Fractures of the long bones HP:0003084

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by nicolevasilevsky** [comment]
HP_0041143
